requirements.txt

```diff
@@ -9,4 +9,4 @@ natsort
 signac>=1.0.0
 watchdog
 webassets>=2.0.0
-werkzeug
+werkzeug>=2.1.0
```

Comment:
Do we still need werkzeug pinned in light of https://github.com/glotzerlab/signac-dashboard/pull/143#discussion_r944563455 ?

Reply:
Good call. I forgot about that. I'll remove the pinning everywhere.

Reply:
Nope. We have explicit imports from `werkzeug`. We should keep it as an explicit dependency, rather than a transitive one via `flask`.

https://github.com/glotzerlab/signac-dashboard/blob/ac20abdf05d93bc65a20022ff68aa06cf3d89bf2/signac_dashboard/util.py#L6

https://github.com/glotzerlab/signac-dashboard/blob/0fd1bb9aece7db6ee70eeaa3f0868f09f27bc5cc/signac_dashboard/dashboard.py#L25

Reply:
Okay!